OoyalaSkinSampleApp/app/build.gradle

```diff
@@ -54,13 +64,7 @@ task copyTask(type: Copy) {
 tasks.copyTask.execute()
 
 dependencies {
-    compile 'com.google.android.gms:play-services-ads:+'
-    compile files('libs/FWAdManager.jar')
-    compile files('libs/OoyalaFreewheelSDK.jar')
-    compile files('libs/OoyalaIMASDK.jar')
-    compile files('libs/OoyalaSDK.jar')
-    compile files('libs/ima-android-v313.jar')
-
-    compile files('libs/OoyalaSkinSDK.jar')
```

Comment:
What if imc-android-v313.jar change name again? Anyway this is not a must and I can revert it back.